pulpanalytics/signals.py

```diff
@@ -0,0 +1,18 @@
+from django.db.models.signals import pre_save
+from django.dispatch import receiver
+from django.utils import timezone
+
+from pulpanalytics.models import System
+
+
+@receiver(pre_save, sender=System)
+def _system_pre_save(sender, instance, raw, **kwargs):
+    if not raw:
```

Comment:
What does this if statement do, can a comment be added please?

Reply:
https://docs.djangoproject.com/en/4.1/ref/signals/#pre-save